File: joseffe-barroso/Python/aula03/exercise11.py
rectangleHeight = float(input('Digite a altura do retângulo: '))
rectangleBase = float(input('Digite a base do retângulo: '))

totalArea = rectangleBase * rectangleHeight

if (totalArea > 100):
    print(f"O Terreno é grande, e possui: {totalArea:.2f} m²")
else:
    print("Fim do programa")

File: joseffe-barroso/Java/src/entrada_processamento_saida/aula03/ddd/Exercise11.java
package ddd;

import java.util.Scanner;

public class Exercise11 {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);

		double rectangleHeight, rectangleBase, totalArea;

		System.out.printf("Digite a altura do retângulo: ");
		rectangleHeight = scanner.nextDouble();

		System.out.printf("Digite a base do retângulo: ");
		rectangleBase = scanner.nextDouble();

		totalArea = rectangleBase * rectangleHeight;

		if (totalArea > 100) {
			System.out.printf("O Terreno é grande, e possui: %f m²", totalArea);
		} else {
			System.out.printf("Fim do programa");
		}

		scanner.close();
	}

}
